app/views/shared/_form_submit_button.html.erb

```diff
@@ -0,0 +1,5 @@
+<% unless ENV['DISABLE_SANDBOX_WARNING'] == 'true' %>
+  <%= form.submit value: 'Submit test request for approval', class: 'form-button' %>
+<% else %>
+  <%= form.submit class: 'form-button' %>
+<% end %>
```

Comment:
Surprised this didn't get caught by Rubocop (I guess it doesn't read ERB?), but `if/else` is preferred to `unless/else`.